Project.toml

```diff
@@ -18,7 +18,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 [compat]
 CuArrays = "^1, ^2"
 FFTW = "^1"
-FourierFlows = "^0.4"
+FourierFlows = "≥0.4.5, ^0.4"
```

Comment:
nice

Reply:
yeap; that's the version with the updated `plan_flows_rfft`.